.releaserc

```diff
@@ -0,0 +1,27 @@
+{
+  "branches": ["main"],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogFile": "CHANGELOG.md"
+      }
+    ],
+    [
+      "@semantic-release/github",
+      {
+        "assets": [{ "path": "dist", "label": "package distribution" }]
```

Comment:
how about adding the `addReleases` also?

Reply:
Sounds cool, what should be the value for `addReleases` ? Can be false, "bottom" or "top"

Reply:
added the config as "addReleases": "top"